src/Sylius/Bundle/CoreBundle/Message/Admin/ResendOrderConfirmationEmail.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Message\Admin;
```

Comment:
Same question as in parallel PR https://github.com/Sylius/Sylius/pull/15715#discussion_r1447817352

CoreBundle doesn't depend or know about any admins, so `Admin` cannot be part of namespace.